app/jobs/cache/browser_channels/main.rb

```diff
@@ -0,0 +1,30 @@
+class Cache::BrowserChannels::Main
+  include Sidekiq::Worker
+  sidekiq_options queue: :scheduler, retry: false
+
+  LAST_RAN_AT_KEY = "cache_browser_channels_main_last_ran_at".freeze
+  RESPONSES_PREFIX_LENGTH = 3
```

Comment:
Why is this 3? Three hex chars will result in around ~170 channels per response, whereas 4 hex chars will result in about 11. Are we trying to limit the number of files generated?